### PR TITLE
[enhancement](binlog) Support gc binlogs by history nums and size

### PR DESCRIPTION
**Pull request by w41ter**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by w41ter** [comment]
run buildall

**Comment by w41ter** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/binlog/TableBinlog.java [hunk 1]
```diff
@@ -305,10 +338,16 @@ public void getBinlogInfo(Database db, BaseProcResult result) {
             info.add(lastBinlogCommittedTime);
             info.add(readableLastBinlogCommittedTime);
             String binlogTtlSeconds = null;
+            String binlogMaxBytes = null;
+            String binlogMaxHistoryNums = null;
             if (binlogConfig != null) {
                 binlogTtlSeconds = String.valueOf(binlogConfig.getTtlSeconds());
+                binlogMaxBytes = String.valueOf(binlogConfig.getMaxBytes());
```
Comment (dataroaring) [inline fe/fe-core/src/main/java/org/apache/doris/binlog/TableBinlog.java:345]: Should we set a default max bytes as 5% of xmx?

Comment (w41ter) [reply fe/fe-core/src/main/java/org/apache/doris/binlog/TableBinlog.java:345]: Based on the known information, it seems that using a time-based gc is already effectively controlling memory usage. Directly limiting memory usage to a maximum of only 5% would introduce other variables and compromise stability. It is recommended to maintain consistency with previous behavior for now.

**Comment by w41ter** [comment]
run performance

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**